tests/integration_test/integration_test.sh

```diff
@@ -204,264 +226,624 @@ else
     INGEST_URL="https://api.ingest.${ENV}.data.humancellatlas.org/"
 fi
 
+print_style "info" "getting version"
+print_style "info" "$(date +"%Y-%m-%d %H:%M:%S")"
 GCS_ROOT="gs://${GCLOUD_PROJECT}-cromwell-execution/caas-cromwell-executions"
 
 function get_version {
-  REPO=$1
-  VERSION=$2
-  BASE_URL="https://api.github.com/repos/HumanCellAtlas/${REPO}"
-  BRANCHES_URL="${BASE_URL}/branches/${VERSION}"
-  COMMITS_URL="${BASE_URL}/commits/${VERSION}"
-
-  STATUS_CODE=$(curl -s -o /dev/null -w "%{http_code}" "${BRANCHES_URL}")
-
-  if [ "${STATUS_CODE}" != "200" ]; then
-    STATUS_CODE=$(curl -s -o /dev/null -w "%{http_code}" "${COMMITS_URL}")
-  fi
-
-  if [ "${STATUS_CODE}" != "200" ]; then
-    # 1>&2 prints message to stderr so it doesn't interfere with return value
-    print_style "warn" "Couldn't find ${REPO} branch or commit ${VERSION}. Using master instead." 1>&2
-    echo "master"
-  else
-    echo "${VERSION}"
-  fi
+    REPO=$1
+    VERSION=$2
+    BASE_URL="https://api.github.com/repos/HumanCellAtlas/${REPO}"
+    BRANCHES_URL="${BASE_URL}/branches/${VERSION}"
+    COMMITS_URL="${BASE_URL}/commits/${VERSION}"
+
+    STATUS_CODE=$(curl -s -o /dev/null -w "%{http_code}" "${BRANCHES_URL}")
+
+    if [ "${STATUS_CODE}" != "200" ]; then
+        STATUS_CODE=$(curl -s -o /dev/null -w "%{http_code}" "${COMMITS_URL}")
+    fi
+
+    if [ "${STATUS_CODE}" != "200" ]; then
+        # 1>&2 prints message to stderr so it doesn't interfere with return value
+        print_style "warn" "Couldn't find ${REPO} branch or commit ${VERSION}. Using master instead." 1>&2
+        echo "master"
+    else
+        echo "${VERSION}"
+    fi
 }
 
-# 1. Clone Lira if needed
-if [ "${LIRA_MODE}" == "github" ] || [ ${LIRA_MODE} == "image" ];
+function clone_secondary_analysis_repo {
+    print_style "info" "Cloning secondary analysis deploy repo"
+    git clone git@github.com:HumanCellAtlas/secondary-analysis-deploy.git
+
+    export SECONDARY_ANALYSIS_DIR="${PWD}/secondary-analysis-deploy"
+    cd "${SECONDARY_ANALYSIS_DIR}"
+
+    if [ "${SECONDARY_ANALYSIS_VERSION}" == "latest_released" ];
```

Comment:
We don't have tagged releases of the `secondary-analysis-deploy` repo...so I think it makes sense to just use the master branch here

Reply:
True - this may have been premature optimization. I'll remove it.